exercises/triangle/spec/triangle_spec.cr

```diff
@@ -2,59 +2,79 @@ require "spec"
 require "../src/*"
 
 describe "Triangle" do
-  it "true if all sides are equal" do
+  it "all sides are equal" do
     Triangle.new([2, 2, 2]).equilateral?.should eq(true)
   end
 
-  pending "false if any side is unequal" do
+  pending "any side is unequal" do
     Triangle.new([2, 3, 2]).equilateral?.should eq(false)
   end
 
-  pending "false if no sides are equal" do
+  pending "no sides are equal" do
     Triangle.new([5, 4, 6]).equilateral?.should eq(false)
   end
 
-  pending "All zero sides are illegal, so the triangle is not equilateral" do
+  pending "all zero sides is not a triangle" do
     Triangle.new([0, 0, 0]).equilateral?.should eq(false)
   end
 
-  pending "true if last two sides are equal" do
+  pending "sides may be floats" do
+    Triangle.new([0.5, 0.5, 0.5]).equilateral?.should eq(true)
```

Comment:
as discussed in individual commit message, note test case addition. Since now there may be `Array(Float64)`, consider the student experience when implementing, whereas before there was only `Array(Int32)`. See that the example had to change. If it is too hard for the student to implement this, consider the alternative of changing the generator to not emit these tests.